src/Compilers/VisualBasic/Portable/Symbols/Source/SourceMemberContainerTypeSymbol.vb

```diff
@@ -3808,6 +3815,33 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
         End Sub
 
+        Private Sub CheckTupleNamesAgainstBase(type As NamedTypeSymbol, diagnostics As DiagnosticBag)
```

Comment:
In VB the error reporting is a bit more sophisticated (better location) so there is a difference. The VB method uses `Me.GetInheritsOrImplementsLocation`.